tests/plan.bats

```diff
@@ -201,15 +201,23 @@ setup() {
 
 }
 
-@test "plan/client/env default" {
+@test "plan/client/env optional set" {
   cd "${TESTDIR}"
 
   export TEST_DEFAULT="hello universe"
+  export TEST_OPT_DEFAULT="hello universe"
+  export TEST_OPTIONAL="foobar"
 
-  "$DAGGER" "do" -p ./plan/client/env/default.cue test
+  "$DAGGER" "do" -p ./plan/client/env/optional.cue test set
```

Comment:
Might be good `assert_success` here?

Reply:
Now that you say that I remember something about it. Can you remember me why it’s needed?

Reply:
Without `run` I think this is enough. A failure in that command will fail the test.

Reply:
👍🏻